ui/shared/ConfirmationDialog.qml

```diff
@@ -26,6 +27,7 @@ ModalPopup {
 
     Text {
```

Comment:
StyledText handled the color and the font so you don't have to
```suggestion
    StyledText {
```

Reply:
Dang! Shoulda noticed that! Fixed!